pyro/contrib/mue/statearrangers.py

```diff
@@ -0,0 +1,152 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+import torch
+import torch.nn as nn
+
+
+def mg2k(m, g):
+    """Convert from (m, g) indexing to k indexing."""
+    return 2*m + 1 - g
+
+
+class profile(nn.Module):
+
```

Comment:
Add a docstring, including:
- meaning, type, and shape of each input parameter (see other Pyro classes for example sphinx notation)
- space and time complexity using `O(-)` notation, and
- one or two citations.